CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 CHANGELOG
 =========
 
-0.2.7
+1.0.0
```

Comment:
I don't think that's a good idea because the protocol itself is still an Internet Draft.

Reply:
OK, reverted